presto-docs/src/main/sphinx/admin/properties.rst

```diff
@@ -356,6 +368,30 @@ Spilling Properties
     Enables using a randomly generated secret key (per spill file) to encrypt and decrypt
     data spilled to disk
 
+``experimental.spiller.single-stream-spiller-choice``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+    * **Type:** ``String``
+    * **Default value:** ``LOCAL_FILE``
+
+    The Single Stream Spiller to be used when spilling is enabled. There are two options
+    LOCAL_FILE (default) and TEMP_STORAGE.
+
+``experimental.spiller.spiller-temp-storage``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+    * **Type:** ``String``
+    * **Default value:** ``local``
+
+    Temp storage used by spiller when single stream spiller is set to TEMP_STORAGE
+
+``experimental.temp-storage-buffer-size``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+    * **Type:** ``Data Size``
+    * **Default value:** ``4KB``
+
+    Value of buffer size when Single Stream Spiller is set to TEMP_STORAGE
```

Comment:
s/ Value of buffer size when Single Stream Spiller / Size of buffer when `experimental.spiller.single-stream-spiller-choice`

Reply:
Done